former-developers.md

```diff
@@ -0,0 +1,19 @@
+The SuperDARN Radar Software Toolkit (RST) was originally written by R.J Barnes and K. Baker at the Johns Hopkins University Applied Physics Laboratory (JHUAPL). Since then, members of the international SuperDARN community have contributed to the RST by adding new features, writing documentation, improving modularity, refactoring, fixing bugs, and testing the software. Below is a list of contributors to the RST. If you believe someone is missing from this list, please contact the Data Analysis Working Group <darn-dawg@isee.nagoya-u.ac.jp>.
+
+- D. Andre, `fitacf`
+- K. Baker, original JHUAPL author
+- R.J. Barnes, original JHUAPL author
+- D.D. Billett, documentation, testing
+- E.C. Bland, DAWG Co-Chair (2019-), documentation, testing
+- W.A. Bristow, `make_grid`
+- A.G. Burrell, documentation, bugfixes, testing
+- R.A. Greenwald, `fitacfex2`
```

Comment:
```suggestion
- R.A. Greenwald, `fitacfex`
```

While Ray's contributions are probably immeasurable, he and Kjellmar developed the original `fitacfex` algorithm while AJ developed `fitacfex2` (with Ray's input I imagine)